plugins/coverimage.koplugin/main.lua

```diff
@@ -0,0 +1,268 @@
+                    sample_input:onShowKeyboard()
+                end,
+            },
+            -- menu entry: enable
+            {
+                text = _("Save current book cover as screensaver image"),
+                checked_func = function()
+                    return self:_enabled() and pathOk(self.cover_image_path)
+                end,
+                enabled_func = function()
+                    return self.cover_image_path ~= "" and pathOk(self.cover_image_path)
+                end,
+                callback = function()
+                    if self.cover_image_path ~= "" then
+                        self.enabled = not self.enabled
+                        G_reader_settings:saveSetting("cover_image_enabled", self.enabled)
+                        if self.enabled then
+                            self:onReaderReady(self.ui.doc_settings)
+                        else
+                            self:cleanUpImage()
+                        end
+                    end
+                end,
+            },
+            -- menu entry: exclude this cover
+            {
+                text = _("Exclude this book cover"),
+                checked_func = function()
+                    return self.ui and self.ui.doc_settings and self.ui.doc_settings:readSetting("exclude_cover_image") == true
+                end,
+                callback = function()
+                    if self.ui.doc_settings:readSetting("exclude_cover_image") == true then
+                        self.ui.doc_settings:saveSetting("exclude_cover_image", false)
+                        self:onReaderReady(self.ui.doc_settings)
+                    else
+                        self.ui.doc_settings:saveSetting("exclude_cover_image", true)
+                        self:cleanUpImage()
+                    end
+                    self.ui:saveSettings()
+                end,
+                separator = true,
+            },
+            -- menu entry: set fallback image
+            {
+                text = _("Set fallback image when no cover or excl. book"),
+                checked_func = function()
+                    return lfs.attributes(self.cover_image_fallback_path, "mode") == "file"
+                end,
+                help_text =  _("File to use as cover image, when no cover is wanted or book is excluded.\nLeave it blank to use nothing."),
+                keep_menu_open = true,
+                callback = function(menu)
+                    local InputDialog = require("ui/widget/inputdialog")
+                    local sample_input
+                    sample_input = InputDialog:new{
+                        title = _("Filename of fallback image"),
+                        input = self.cover_image_fallback_path,
+                        input_type = "string",
+                        description = _("You can enter the filename of the fallback image here.\n" ..
+                            "Leave it empty to clean up on close."),
```

Comment:
For translation with multiline use single line with `\n` or `[[]]`, no concat.
```suggestion
                        description = _("Leave this empty to remove the cover when the document is closed."),
```